unit/ansi-c/expr2c.cpp

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************\
+
+Module: Unit test for expr2c
+
+Author: Diffblue
+
+\*******************************************************************/
+
+#include <ansi-c/expr2c.h>
+#include <testing-utils/use_catch.h>
+#include <util/arith_tools.h>
+#include <util/bitvector_expr.h>
+#include <util/bitvector_types.h>
+#include <util/config.h>
+#include <util/namespace.h>
+#include <util/symbol_table.h>
+
+TEST_CASE("rol_2_c_conversion_unsigned", "[core][ansi-c][expr2c]")
+{
+  auto lhs = from_integer(31, unsignedbv_typet(32));
```

Comment:
Personal preference here :-) But for tests used for bit-twiddling operations, I'd be rather tempted to use hex representations of the various immediate constants in tests like this. Might just be my background, but I find that easier to mentally model, rather than also adding a decimal->hex->binary mental translation :-)